core/src/main/res/values/themes.xml

```diff
@@ -11,7 +11,7 @@
     <!--colorPrimary colors map to components and elements, such as app bars and buttons. -->
     <!--colorSecondary colors are most often used as accents on components, such as FABs and -->
     <!--selection controls.-->
-    <item name="colorPrimary">@color/color_primary</item>
+    <item name="colorPrimary">@color/denim_blue800</item>
```

Comment:
these parts where we specify a color specifically should be moved into `KiwixTheme` but we should leave the ones that reference attributes in the BaseTheme